### PR TITLE
Ios lite simple app improvment

### PR DESCRIPTION
**Pull request by gorBaghdasaryan**
add photo selection and image preview for tensorflow lite simple app

### CODE REVIEW

**Comment by rmlarsen** [comment]
@gorBaghdasaryan your PR contains a bunch of XCode files. Did you submit those by mistake?

**Comment by gorBaghdasaryan** [comment]
@rmlarsen no, not by chance. all these changes are needed

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @petewarden: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @petewarden: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @petewarden, @miaout17: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/contrib/lite/examples/ios/simple/AppDelegate.m [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Copyright 2015 Google Inc. All rights reserved.
```
Comment (miaout17) [inline tensorflow/contrib/lite/examples/ios/simple/AppDelegate.m]: Could you change this to...

> // Copyright 2018 The TensorFlow Authors. All Rights Reserved.


Thanks

File: tensorflow/contrib/lite/examples/ios/simple/ios_image_load.h [hunk 2]
```diff
@@ -15,9 +15,7 @@
 #ifndef TENSORFLOW_CONTRIB_LITE_EXAMPLES_IOS_SIMPLE_IOS_IMAGE_LOAD_H_
 #define TENSORFLOW_CONTRIB_LITE_EXAMPLES_IOS_SIMPLE_IOS_IMAGE_LOAD_H_
 
+#import <UIKit/UIKit.h>
 #include <vector>
 
-std::vector<uint8_t> LoadImageFromFile(const char* file_name, int* out_width,
-                                       int* out_height, int* out_channels);
-
-#endif  // TENSORFLOW_CONTRIB_LITE_EXAMPLES_IOS_SIMPLE_IOS_IMAGE_LOAD_H_
```
Comment (miaout17) [inline tensorflow/contrib/lite/examples/ios/simple/ios_image_load.h]: Missing `#ifdef` here. This doesn't compile

File: tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 Google Inc. All rights reserved.
+// Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```
Comment (miaout17) [inline tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm]: // Copyright 2018 The TensorFlow Authors. All Rights Reserved.

Thanks!

**Review by miaout17 — CHANGES_REQUESTED** [review]

**Comment by petewarden** [comment]
Sorry for the slow response on this. It looks like a good improvement to the sample app, but one of our goals is to keep the code for the example as simple as possible, so it might be best as a separate github project. If you do create one, we'd be happy to link to it from documentation.

**Pull request closed without merging**